hms-api/src/rest/HackathonPreviewResponse.ts

```diff
@@ -1,34 +1,37 @@
 /* eslint-disable require-jsdoc */
 
-import Uuid from '../util/Uuid';
```

Comment:
the backend only uses single quotes for strings (it will actually fail lint checks with double quotes)